manifests/install.pp

```diff
@@ -35,6 +35,7 @@
     archive { $real_duply_archive_name:
       ensure           => $real_duply_package_ensure,
       url              => $real_duply_archive_url,
+      proxy_server     => $duplicity::duply_archive_proxy,
```

Comment:
In order to use this parameter, the version of the archive module must be 0.5.0 or higher. Can you please update the `metadata.json` (`>= 0.5.0 < 1.0.0`), the `.fixtures.yaml` and the `spec_helper_acceptance.rb` files?